selfdrive/car/hyundai/carstate.py

```diff
@@ -29,6 +29,8 @@ def __init__(self, CP):
                           "GEAR_SHIFTER"
     if CP.carFingerprint in CANFD_CAR:
       self.shifter_values = can_define.dv[self.gear_msg_canfd]["GEAR"]
+    elif self.CP.carFingerprint in (HYBRID_CAR | EV_CAR):
+      self.shifter_values = can_define.dv["ELECT_GEAR"]["Elect_Gear_Shifter"]
     elif self.CP.carFingerprint in CAN_GEARS["use_cluster_gears"]:
       self.shifter_values = can_define.dv["CLU15"]["CF_Clu_Gear"]
     elif self.CP.carFingerprint in CAN_GEARS["use_tcu_gears"]:
```

Comment:
remove `# preferred and elect gear methods use same definition`comment below

Reply:
@sshane Updated.